ts/demo-functions/src/read_yaml_test.ts

```diff
@@ -51,17 +52,27 @@ describe('readYaml', () => {
       expectedIntermediateFile,
       FileFormat.YAML
     );
-    functionConfig.data![SOURCE_DIR] = sourceDir;
+    functionConfig.data = {
```

Comment:
Any reason to prefer directly setting data vs previous way of "functionConfig.data![SOURCE_DIR] = sourceDir;"

Reply:
Yes, this is more readable/elegant way to set objects with variable key names. We're not inheriting any existing keys.